packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -621,15 +628,42 @@ class StackViewport extends Viewport implements IStackViewport {
   }
 
   /**
-   * Sets the properties for the viewport on the default actor. Properties include
-   * setting the VOI, inverting the colors and setting the interpolation type, rotation
+   * Update the default properties of the viewport and add properties by imageId if specified
```

Comment:
it requires documentation that IF the imageId/index is not provided it will be set for all